### PR TITLE
fix: Correct quick start in README

### PR DESCRIPTION
**Pull request by saulmc**
Updates outdated reference in quick start

### CODE REVIEW

**Review by elisealix22 — APPROVED** [review]
Thank you!

File: README.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ const messages = await conversation.messages()
 await conversation.send('gm')
 // Listen for new messages in the conversation
 for await (const message of await conversation.streamMessages()) {
-  console.log(`[${message.senderAddress}]: ${message.text}`)
+  console.log(`[${message.senderAddress}]: ${message.content}`)
```
Comment (jhaaaa) [inline README.md:90]: Thank you for this @saulmc! There is one more instance of `message.text` on line 217 - just making sure - shall we update that one too? =)

Comment (neekolas) [reply README.md:90]: Good eye

Comment (jhaaaa) [reply README.md:90]: Tech writer habit 😅

Comment (saulmc) [reply README.md:90]: oooh nice one @jhaaaa!

**Pull request merged**